Cubical/Foundations/CartesianKanOps.agda

```diff
@@ -5,6 +5,7 @@ module Cubical.Foundations.CartesianKanOps where
 open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.GroupoidLaws
 open import Cubical.Foundations.Transport
```

Comment:
I think with your changes, these imports are not needed anymore:

```suggestion
```

Reply:
I guess I can commit this and we see if the CI is happy